public/css/main.css

```diff
@@ -1,13 +1,19 @@
+html {
+	height: 100%;
+}
+
 body {
-	/* margin: 0 auto; */
-	/* width: 600px; */
+	display: flex;
+		align-items: center;
+		justify-content: center;
```

Comment:
Ahh right. No problem. Go ahead and experiment first. I was just wondering about it only. :stuck_out_tongue_closed_eyes: Yup you gonna tackle the issue of modularity in this PR?